cms/templates/visibility_editor.html

```diff
@@ -43,96 +40,50 @@ <h4 class="title">${_('No content groups exist')}</h4>
-                        </div>
+            <h3 class="modal-section-title" id="visibility-title">${_('Make visible to:')}</h3>
+                <div class="modal-section-content partition-visibility">
+                <select aria-labelledby="visibility-title">
+                    <option value="-1" ${'selected="selected"' if selected_partition_index == -1 else ''}>
+                        ${_('All Learners and Staff')}
+                    </option>
+                    % for index, partition in enumerate(selectable_partitions):
+                        <option value="${partition["id"]}" id="visibility-partition-${partition["id"]}" ${'selected="selected"' if selected_partition_index == index else ''}>
+                            ${partition["name"]}
+                        </option>
+                    % endfor
+                </select>
 
-                        <div class="field field-radio field-visibility-level">
-                            <input type="radio" id="visibility-level-specific" name="visibility-level" value="" class="input input-radio visibility-level-specific"  ${'checked="checked"' if has_selected_groups else ''} />
-                            <label for="visibility-level-specific" class="label">${_('Specific Content Groups')}</label>
+                % for index, partition in enumerate(selectable_partitions):
+                    <div class="partition-group-control partition-group-control-${partition["id"]} ${'is-hidden' if selected_partition_index != index else ''}">
+                        <div class="partition-group-directions" id="partition-group-directions-${partition["id"]}">${_('Select one or more options:')}
+                        % for group in partition["groups"]:
+                            <div role="group" aria-labelledby="visibility-partition-${partition["id"]}" aria-describedby="partition-group-directions-${partition["id"]}"
+                                class="field partition-group-visibility partition-group-visibility-${partition["id"]} ${'was-removed' if group["deleted"] else ''}">
+                                <input type="checkbox"
+                                    id="visibility-group-${partition["id"]}-${group["id"]}"
+                                    name="visibility-group"
+                                    value="${group["id"]}"
+                                    class="input input-checkbox"
+                                    ${'checked="checked"' if group["selected"] else ''}
+                                />
+                                % if group["deleted"]:
+                                <label for="visibility-group-${partition["id"]}-${group["id"]}" class="label">
+                                    ${_("Deleted Group")}
+                                    <span class="note">${_('This group no longer exists. Please choose another or allow access to All Learners and Staff.')}</span>
```

Comment:
"This group no longer exists. Choose another group or make this component visible to All Learners and Staff."